src/modules/api/auth/decorators/index.ts

```diff
@@ -0,0 +1,11 @@
+import { createParamDecorator, ExecutionContext } from "@nestjs/common";
+
+export const ClientData = createParamDecorator(
```

Comment:
The decorator should live in the User Module as it contains some information about the user. Also, it should decorate an object instead of a string so that it can be easily scaled